easybuild/easyconfigs/c/COBRApy/COBRApy-0.26.0-foss-2021a.eb

```diff
@@ -0,0 +1,99 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'COBRApy'
+version = '0.26.0'
+
+homepage = 'https://opencobra.github.io/cobrapy/'
+description = """COBRApy is a package for constraint-based modeling of metabolic networks."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python',            '3.9.5'),
+    ('ruamel.yaml',       '0.17.21'),
+    ('SciPy-bundle',      '2021.05'),
+    ('sympy',             '1.8'),
+    ('SWIG',              '4.0.2'),
```

Comment:
@smoretti `SWIG` should only be a build dependency I think?

Reply:
Should be